dimagi/utils/logging.py

```diff
@@ -29,8 +29,11 @@ def notify_error(message):
 
 
 def notify_exception(request, message=None, details=None):
+    if request is not None:
+        message = message or request.path
     notify_logger.error(
-        'Notify Exception: %s' % (message or request.path),
+        'Notify Exception: %s' % (message
+                                  or "No message provided, fix error handler"),
```

Comment:
Nope, it wouldn't cause an error because it was referencing `message or request.path`, so as long as message is not None (which is the case in that link) then request.path never even gets evaluated.